committee-steering/governance/wg-governance.md

```diff
@@ -0,0 +1,103 @@
+# Kubernetes Working Group Formation and Disbandment
+
+Draft 1.0 // Jaice Singer DuMars // June, 2018
```

Comment:
Not sure we want to get in the habit of having docs like this owned by folks. Perhaps to give a nod to Jaice we have a "doc history" section at the end with significant versions or references.  But there is also git history.

Reply:
#2814